src/components/ArticleList.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { getArticles } from "../api";
+import { useState, useEffect } from "react";
+
+export default function ArticleList({filterTopic}) {
+    const [articles, setArticles] = useState([]);
+    useEffect(() => {
+        getArticles(filterTopic)
+            .then(data => {
+                setArticles(data.articles); 
+            })
+            .catch(error => {
+                console.error('Error fetching articles:', error);
+            });
+    }, [filterTopic]);
+    return (
+        <main>
+            <h2> Articles </h2>
+            { articles.map((article, index) => 
+            (
+                <div key={index}>
+                    <h3>{article.title}</h3>
+                    <img src={article.article_img_url} alt={article.title} />
+                </div>
+            ))}
```

Comment:
We've got loads more data available than just the title and image, why not take advantage of this and show more of them to give the user more info?